jupyter_ydoc/ynotebook.py

```diff
@@ -220,9 +220,9 @@ def get(self) -> Dict:
 
         return dict(
             cells=cells,
-            metadata=meta["metadata"],
-            nbformat=int(meta["nbformat"]),
-            nbformat_minor=int(meta["nbformat_minor"]),
+            metadata=meta.get("metadata", {}),
+            nbformat=int(meta.get("nbformat", 0)),
+            nbformat_minor=int(meta.get("nbformat_minor", 0)),
```

Comment:
Should it default to format 4.5 instead of 0.0? Or will it be immediately overridden?

Reply:
Thanks for the comment @krassowski. I will look into it and change it.

Reply:
@hbcarlos thanks for proposing looking into the matter. One comment here as it impacts 3 packages, the improvement should use default version imported from a single source of truth (best candidate is this one) and not hard code value as the default nbformat may change. We could also add a static method to `YNotebook` to test if the document content matches an empty notebook.